nemo/deploy/nlp/query_llm.py

```diff
@@ -259,7 +261,17 @@ def query_llm(
                     return "Unknown output keyword."
 
                 sentences = np.char.decode(output.astype("bytes"), "utf-8")
-                return sentences
+                if openai_format_response:
```

Comment:
Do you think we can do this openai response conversion in the here https://github.com/NVIDIA/NeMo/blob/athitten/openai_format_response/nemo/deploy/service/rest_model_api.py#L101 ?

Also, is there any advantage of supporting the openai response type at the class level? Maybe our default should be the openai response format (still not sure though).

Reply:
@oyilmaz-nvidia yes we can do the openAI response conversion in `rest_model_api.py` after output is received [here](https://github.com/NVIDIA/NeMo/blob/athitten/openai_format_response/nemo/deploy/service/rest_model_api.py#L101). Do you recommend that ?

Also regarding having OpenAI format as the default, do any of the methods/functions in the deploy module or any downstream tasks rely on output being returned directly (non OpenAI format) i.e the way it is now ? I wasn't sure of this so added it as an option to not break anything.